style.css

```diff
@@ -0,0 +1,186 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: rgb(56, 56, 56);
+    justify-content: center;
+}
+
+.main-grid {
+    display: grid;
+    grid-template: 100% / 400px 700px 400px;
+}
+
+.navigation {
+    display: grid;
+    padding: 50px;
+    color: white;
+    gap: 20px;
+}
+
+.navigation img {
+    width: 100px;
+    height: 100px;
+}
+
+.navigation h4 {
+    font-size: 30px;
+    font-weight: 400;
+}
+
+.navigation h4:hover {
+    color: orange
+}
+
+.navigation h3 {
+    font-size: 30px;
+    font-weight: 400;
+    border-radius: 50px;
+    border-width: 2px;
+    border-color: white;
+    border-style: solid;
+    text-align: center;
+    padding: 10px;
+}
+
+.middle {
+    color: white;
+    border-width: 2px;
+    border-color: rgb(82, 82, 82);
+    border-style: solid;
+}
+
+.middle h1 {
+    font-size: 40px;
+    padding: 20px;
+}
+
+.middle-top {
+    display: grid;
+    grid-template: 42px / 80px 612px;
+    padding: 20px;
+    gap: 5px
+}
+
+.middle-top img {
+    width: 55px;
+    border-radius: 50px;
+    margin-left: 20px;
+}
+
+.middle-top input {
+    border: 2px solid rgb(82, 82, 82);
+    font-size: 25px;
+    height: 60px;
+    padding-left: 10px;
+    border-top-left-radius: 5px;
+}
+
+.icons-and-tweet-button {
+    display: grid;
+    grid-template: 50px / 200px 120px;
```

Comment:
Maybe the Home section could of been further broken up into sections for the borders? So they dont clash as much?